enclave/CMakeLists.txt

```diff
@@ -78,4 +79,11 @@ install_enclaves(
 
 if (WITH_EEID)
   enclave_compile_definitions(oeenclave PRIVATE OE_WITH_EXPERIMENTAL_EEID)
+  # oeenclave currently does not link against the oecrypto* library by default.
+  # Consequently, it does not inherit the include paths of the crypto (e.g., mbedtls)
+  # headers, which are needed by eeid.c. Therefore, we add the paths here.
+  enclave_include_directories(
```

Comment:
Shouldn't this be openssl and mbedtls headers?

Reply:
EEID implementation currently is based on mbedtls only.